integration/tests/lifecycle_nitrorpc_v04.test.ts

```diff
@@ -353,9 +386,12 @@ describe('nitrorpc_v04 lifecycle', () => {
 
         const resizeReceipt = await blockUtils.waitForTransaction(resizeChannelTxHash);
         expect(resizeReceipt).toBeDefined();
+
+        const postCloseAccountBalance = await bobClient.getAccountBalance(CONFIG.ADDRESSES.USDC_TOKEN_ADDRESS);
+        expect(postCloseAccountBalance).toBe(toRaw(finalBobAmount)); // 1125
     });
 
-    it('should close channel and withdraw with app funds', async () => {
+    it('should close Bob channel', async () => {
```

Comment:
Do I understand correctly that you have decided to showcase that 2 funds withdrawal cases are possible:
1. resize + close (Alice channel)
2. just close (Bob channel)
?